packages/webview_flutter/webview_flutter_platform_interface/lib/src/v4/types/javascript_message.dart

```diff
@@ -0,0 +1,17 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:meta/meta.dart';
+
+/// A message that was sent by JavaScript code running in a [WebView].
+@sealed
```

Comment:
This sealed can be removed too. The `WKWebView` implementation may add additional parameters from [WKScriptMessage](https://developer.apple.com/documentation/webkit/wkscriptmessage?language=objc).